tests/unit_tests/pyanaconda_tests/modules/boss/test_copy_logs_task.py

```diff
@@ -0,0 +1,191 @@
+#
+# Copyright (C) 2021  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import unittest
+from unittest.mock import call, patch
+from pyanaconda.modules.boss.installation import CopyLogsTask
+
+
+class CopyLogsTaskTest(unittest.TestCase):
+    @patch("pyanaconda.modules.boss.installation.glob.glob")
+    @patch("pyanaconda.modules.boss.installation.util.execInSysroot")
```

Comment:
You shouldn't use `execInSysroot`in the task. It means that the task doesn't always use the given `sysroot` argument. Please, use an alternative.